x-pack/plugins/observability_solution/observability_onboarding/public/application/observability_onboarding_flow.tsx

```diff
@@ -33,74 +29,26 @@ export function ObservabilityOnboardingFlow() {
 
   return (
     <QueryClientProvider client={queryClient}>
```

Comment:
Love the simplification here ❤️

Reply:
Thanks @thomheymann for the great refactor suggestions. ❤️